src/components/Footer.test.js

```diff
@@ -1,6 +1,6 @@
 import { setupRtl } from "component-test-setup";
+import Footer from "./Footer";
 
-import { Footer } from "./Footer";
```

Comment:
There shouldn't be any changes to this file. The import here was correct before.

* `import Footer from "./Footer";` is a _default_ import. It retrieves the object under the name `default` from `"./Footer"`, and stores it in a variable named `Footer`.
* `import { Footer } from "./Footer";` is a _named_ import. It retrieves the object under the name `Footer` from `"./Footer"`, and stores it in a variable named `Footer`.

https://stackoverflow.com/questions/46913851/why-and-when-to-use-default-export-over-named-exports-in-es6-modules